### PR TITLE
Refactor package for new SFMC Source plugin and pom changes.

### PR DESCRIPTION
**Pull request by shiva-cirus**
As discussed with Amit , this is the first PR for SFMC Source plugin. This contains the pom file changes and restructuring package to the respective plugin directories.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -187,39 +197,21 @@
...
     </dependency>
 
-
     <dependency>
       <groupId>com.github.salesforce-marketingcloud</groupId>
       <artifactId>fuelsdk</artifactId>
       <version>${salesforce.marketing.cloud.version}</version>
     </dependency>
-
-    <dependency>
```
Comment (albertshau) [inline pom.xml:197]: why are these getting removed? were they never needed in the first place?

File: pom.xml [hunk 2]
```diff
@@ -187,39 +197,21 @@
...
-      <artifactId>cxf-rt-ws-security</artifactId>
-      <version>${cxf.version}</version>
-    </dependency>
-
     <dependency>
       <groupId>commons-lang</groupId>
       <artifactId>commons-lang</artifactId>
       <version>2.6</version>
     </dependency>
+    <dependency>
```
Comment (albertshau) [inline pom.xml]: why is this being added?

File: pom.xml [hunk 3]
```diff
@@ -234,6 +226,12 @@
       <version>${junit.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
```
Comment (rmstar) [inline pom.xml:224]: Why are we adding a dependency on mockito? Is it being used in tests?

File: pom.xml (hunk 1 above)
Comment (shiva-cirus) [reply pom.xml:197]: They were not needed.  CFX library is already part of the SFMC jar.  Ref: https://github.com/salesforce-marketingcloud/FuelSDK-Java   . Check out the installation section of Readme.  CFX library is only required if you try to build the JAR from the source.

File: pom.xml (hunk 3 above)
Comment (shiva-cirus) [reply pom.xml:224]: Yes its being used in Tests ( which will be included in the 3rd PR)

File: pom.xml (hunk 2 above)
Comment (shiva-cirus) [reply pom.xml]: We were trying to rebuild the SFMC jar and remember adding few jar's during that time and this was one of them. I tried to check any dependency in the code for this library but could not find it and was able to build by removing the dependency.  For now I can remove the dependency JAR and update the PR. Let me know.

Comment (albertshau) [reply pom.xml]: yes, please remove anything that's not needed, as it bloats the jar and can potentially cause conflicts later on

File: pom.xml (hunk 1 above)
Comment (albertshau) [reply pom.xml:197]: Did you make sure removing this didn't break the existing sink? I see that the 1.4.0 and 1.5.0 versions of the fuel library have cxf 3.1.2 as a dependency, whereas this project was using 3.1.9

File: pom.xml (hunk 2 above)
Comment (shiva-cirus) [reply pom.xml]: Done

File: pom.xml (hunk 1 above)
Comment (shiva-cirus) [reply pom.xml:197]: I personally did not test the sink but had Colgate team test it. Let me confirm again with them and get back to you.

Comment (shiva-cirus) [reply pom.xml:197]: @albertshau  - I have forwarded the confirmation from Colgate that the SINK plugin that's built as part of 1.5.0 is working fine. This does not have dependency on the CXF jar's.

**Review by albertshau — APPROVED** [review]
lgtm

**Pull request merged**